airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryAsyncFlush.java

```diff
@@ -84,12 +84,12 @@ public void flush(final StreamDescriptor decs, final Stream<PartialAirbyteMessag
 
   @Override
   public long getOptimalBatchSizeBytes() {
-    // todo(ryankfu): this should be per-destination specific. currently this is for Snowflake.
-    // The size chosen is currently for improving the performance of low memory connectors. With 1 Gi of
-    // resource the connector will usually at most fill up around 150 MB in a single queue. By lowering
-    // the batch size, the AsyncFlusher will flush in smaller batches which allows for memory to be
-    // freed earlier similar to a sliding window effect
-    return 25 * 1024 * 1024;
+    // Chosen arbitrarily (mostly to match legacy behavior). We have no reason to believe a larger
+    // number would be worse.
+    // This was previously set to 25B, which ran into rate-limiting issues:
+    // https://cloud.google.com/bigquery/quotas#standard_tables
+    // > Your project can make up to 1,500 table modifications per table per day
+    return 200 * 1024 * 1024;
```

Comment:
History: In our testing, BQ was ooming at 50 so we dropped it to 25. GC looked a lot better after dropping the value. Sounds like you've done some testing though so this may no longer be the case.

Reply:
do you remember how you got the OOMs? (i.e. what source, how many streams, etc) I can try and repro that experiment

Reply:
Let me do some archaeology. It was either 3 stream Faker or charles benchmark source

Reply:
Lots of the connections have been converted to standard inserts, but the entire suite of sources we used was pretty minimal and contained in [this workspace](https://cloud.airbyte.com/workspaces/419d9f61-1f03-4a4b-aff8-ad8cd8ba9f85/connections)

Reply:
Here's [a slack thread](https://airbytehq-team.slack.com/archives/C05H8UCNCK0/p1692925832611409) with info describing the experiment.

Reply:
sweet, that's an easy test setup. I'll run a 5M faker sync. If that works then I'll feel OK with this PR going live.

Reply:
sync looked good. Threshold + batch size were both correctly 200MB:
```
2023-11-03 21:30:30 destination > 2023-11-03 21:30:30 INFO i.a.c.i.d.DetectStreamToFlush(getNextStreamToFlush):123 - flushing: trigger info: edgao_faker_perf_test - users, time trigger: false , size trigger: true current threshold b: 200 MB, queue size b: 202.99 MB, penalty b: 0 bytes, after penalty b: 202.99 MB
```

```
2023-11-03 21:30:30 destination > 2023-11-03 21:30:30 INFO i.a.c.i.d.FlushWorkers(lambda$flush$1):167 - Flush Worker (7c4ea) -- Batch contains: 288673 records, 200 MB bytes.
```

Sync succeeded, and kube reported memory usage was stable at <200MB:
![image](https://github.com/airbytehq/airbyte/assets/5741425/0621b6a6-9206-4608-86ab-834c3b2aca6c)